service/javadsl/client/src/main/scala/com/lightbend/lagom/internal/javadsl/client/CircuitBreakerModule.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.javadsl.client
+
+import javax.inject.{ Inject, Provider, Singleton }
+
+import akka.actor.ActorSystem
+import com.lightbend.lagom.internal.client.{ CircuitBreakerConfig, CircuitBreakerMetricsProviderProvider, CircuitBreakers }
+import com.lightbend.lagom.internal.javadsl.CircuitBreakersPanelImpl
+import com.lightbend.lagom.internal.spi.CircuitBreakerMetricsProvider
+import com.lightbend.lagom.javadsl.client.CircuitBreakersPanel
+import play.api.inject.{ Binding, Module }
+import play.api.{ Configuration, Environment }
+
+class CircuitBreakerModule extends Module {
+
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
+    Seq(
+      bind[CircuitBreakersPanel].to[CircuitBreakersPanelImpl],
+      // for backward compatibility we still need to provide it for wiring
+      bind[CircuitBreakers].toProvider[CircuitBreakersProvider],
+      bind[CircuitBreakerMetricsProvider].toProvider[CircuitBreakerMetricsProviderProvider]
+    )
+  }
+}
+
+@Singleton
+class CircuitBreakersProvider @Inject() (
+  system:               ActorSystem,
+  circuitBreakerConfig: CircuitBreakerConfig,
+  metricsProvider:      CircuitBreakerMetricsProvider
+) extends Provider[CircuitBreakers] {
+  lazy val get = {
+
+    // only in case some 3rd part lib are still wiring the old one.
+    println( // (no logger available here, so good old println)
+      """
+        | +----------------------------------------------------------------------------------------+
+        | | WATCH OUT!!                                                                            |
+        | | com.lightbend.lagom.internal.client.CircuitBreakers is DEPRECATED                      |
+        | | and will be removed on the next release.                                               |
+        | |                                                                                        |
+        | | Instead you should be using com.lightbend.lagom.javadsl.client.CircuitBreakersPanel.   |
+        | |                                                                                        |
+        | | If you don't understand why you are getting this warning, it's probably because        |
+        | | you are using a 3rd part library that is still wiring this deprecated CircuitBreakers. |
+        | +----------------------------------------------------------------------------------------+
+      """.stripMargin
```

Comment:
Ok, I will make it a one-liner. 

I pushed all the other comments except the Javadoc / Scaladoc for `CircuitBreakersPanel`. I will push it soon.